about.html

```diff
@@ -0,0 +1,52 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>KEEP SUAVE BARBERSHOP</title>
+    <link rel="stylesheet" href="./reset.css">
+    <link rel="stylesheet" href="./all.css">
+    <link rel="stylesheet" href="./about-style.css">
```

Comment:
correct order but you should move your google font links above these. it works because most browsers are fast enough but technically you're trying to load font styles that haven't loaded yet because it's called after these styles.

Reply:
*not a big problem. no need to fix. just fyi

Reply:
This is great to know, thank you!